vmas/scenarios/debug/kinematic_bicycle.py

```diff
@@ -0,0 +1,113 @@
+import typing
+from typing import List
+
+import torch
+
+from vmas import render_interactively
+from vmas.simulator.core import Agent, World, Box
+from vmas.simulator.dynamics.kinematic_bicycle import KinematicBicycleDynamics
+from vmas.simulator.scenario import BaseScenario
+from vmas.simulator.utils import Color, ScenarioUtils
+
+if typing.TYPE_CHECKING:
+    from vmas.simulator.rendering import Geom
+
+class Scenario(BaseScenario):
+    def make_world(self, batch_dim: int, device: torch.device, **kwargs):
+        """
+        Kinematic bicycle model example scenario
+        """
+        self.n_agents = kwargs.get("n_agents", 2)
+        width = kwargs.get("width", 0.1) # Agent width
+        l_f = kwargs.get("l_f", 0.1) # Distance between the front axle and the center of gravity
+        l_r = kwargs.get("l_r", 0.1) # Distance between the rear axle and the center of gravity
+        max_steering_angle = kwargs.get("max_steering_angle", torch.deg2rad(torch.tensor(30.0)))
+
+        # Make world
+        world = World(batch_dim, device, substeps=10)
+
+        for i in range(self.n_agents):
+            if i == 0:
+                # Use the kinematic bicycle model for the first agent
+                agent = Agent(
+                    name=f"agent_{i}",
+                    shape=Box(length=l_f+l_r, width=width),
+                    collide=True,
```

Comment:
```suggestion
                    collide=False,
```
Box on box collisions are quite expensive currently and require some tuning.
I suggest we turn it off for this example.

Reply:
I agree!